validator/validator_test.go

```diff
@@ -135,7 +135,7 @@ func testDummyValidator(reflect.Value) interface{} {
 	return 6
 }
 
-//nolint:gochecknoinits
+//nolint:gochecknoinits // We ignore these for tests
 func init() {
```

Comment:
Maybe this should / could be done in [TestMain](https://golang.org/pkg/testing/#hdr-Main)?

Reply:
Good point, fixed in e97548cb87552a5b17071bd5097d429d82f62fe5